From: Dmitry Pisklov <dpisklov@gmail.com>
Date: Mon, 6 Jan 2014 20:08:21 -0500
Subject: hwdb: remove duplicate entry for Samsung 700Z

keyboardbrightnessup/down are not even real keys.

https://bugs.freedesktop.org/show_bug.cgi?id=72311
(cherry picked from commit 245dc22920d6592e50265c82c8def8e4bbf0e665)
---
 hwdb/60-keyboard.hwdb | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/hwdb/60-keyboard.hwdb b/hwdb/60-keyboard.hwdb
index ccfc734..cf3d1fb 100644
--- a/hwdb/60-keyboard.hwdb
+++ b/hwdb/60-keyboard.hwdb
@@ -918,11 +918,6 @@ keyboard:dmi:bvn*:bvr*:bd*:svn[sS][aA][mM][sS][uU][nN][gG]*:pn*SX20S*:pvr*
  KEYBOARD_KEY_77=f22                                    # Touchpad on
  KEYBOARD_KEY_79=f23                                    # Touchpad off
 
-keyboard:dmi:bvn*:bvr*:bd*:svn[sS][aA][mM][sS][uU][nN][gG]*:pn*700Z*:pvr*
- KEYBOARD_KEY_ba=ejectcd
- KEYBOARD_KEY_96=keyboardbrightnessup
- KEYBOARD_KEY_97=keyboardbrightnessdown
-
 keyboard:dmi:bvn*:bvr*:bd*:svn[sS][aA][mM][sS][uU][nN][gG]*:pn*700T*:pvr*
  KEYBOARD_KEY_ad=leftmeta
 
